pkg/spoke/addon/configuration.go

```diff
@@ -0,0 +1,51 @@
+package addon
+
+import (
+	"fmt"
+
+	addonv1alpha1 "github.com/open-cluster-management/api/addon/v1alpha1"
+)
+
+const (
+	installationNamespaceAnnotation = "installNamespace"
```

Comment:
annotations should be namespaced under your name.  `addon.open-cluster-management.io/installNamespace` for instance.

Reply:
added